core/shared/platform/windows/win_malloc.c

```diff
@@ -22,3 +22,9 @@ os_free(void *ptr)
 {
     free(ptr);
 }
+
+int
+os_dumps_proc_mem_info(char *out, unsigned int size)
```

Comment:
Should also implement API for other platforms, e.g. esp-idf, riot , rt-thread and zephyr, or when we build them with wasm_c_api.c in debug mode, there will be undefined reference linkage issue.